packages/styling/src/styles/theme.ts

```diff
@@ -105,6 +108,25 @@ export function createTheme(theme: IPartialTheme, depComments: boolean = false):
     ...theme.semanticColors
   };
 
+  const typography = theme.typography || DefaultTypography;
```

Comment:
Usually we take the user-provided customizations and mix them on top of the defaults, so the defaults will always be guaranteed to be available:
`const typography = { ...DefaultTypography, ...theme.typography };`